pkg/util/kube/pvc_pv.go

```diff
@@ -434,6 +434,11 @@ func GetPVCAttachingNodeOS(pvc *corev1api.PersistentVolumeClaim, nodeClient core
 	var nodeOS string
 	var scFsType string
 
+	if pvc.Spec.VolumeMode != nil && *pvc.Spec.VolumeMode == corev1api.PersistentVolumeBlock {
```

Comment:
Shouldn't we only check the volume mode when we cannot determine the OS via the annotation just like checking the file system type `if scFsType == "ntfs"`?

Reply:
At present, Windows nodes doesn't support blockMode volumes. Or in another word, pods running in Windows nodes won't be with blockMode volumes.

Therefore, until Kubernetes supports it in a future release, we should always use linux nodes for blockMode volumes.

Reply:
What will happen when Kubernetes supports the Window block mode volumes in future? Although the PVC is attached to a Windows node, the "Linux" is returned as the OS, and the backup pod will run on the Linux node to handle the Windows block mode volumes, is this the expected behavior?

Reply:
blockMode volumes are technically be backupable from either linux node or Windows node.
Running blockMode volumes backups on Windows nodes require further efforts.

Therefore, once Windows block mode volumes are supported, on the one hand, we can run all the backups on linux nodes with no efforts; one the other hand, as the ultimate solution, we will support Windows blockMode volumes backup.